### ISSUE TITLE
unable to install terraform 0.14.9

### ISSUE THREAD

**Issue by ahmad-fakhrozy** [tfutils/tfenv#331]
❯ tfenv install 0.14.9
Installing Terraform v0.14.9
Downloading release tarball from https://releases.hashicorp.com/terraform/0.14.9/terraform_0.14.9_darwin_arm64.zip
curl: (22) The requested URL returned error: 403

Tarball download failed

When I tried to the official repo"https://releases.hashicorp.com/terraform/0.14.9/"
the url for 0.14.9 darwin URL is actually "https://releases.hashicorp.com/terraform/0.14.9/terraform_0.14.9_darwin_amd64.zip"

the tfenv point to "terraform_0.14.9_darwin_arm64.zip" but the official repo "terraform_0.14.9_darwin_amd64.zip"

I'm using mac M1
❯ tfenv --version
tfenv 2.2.3

**Comment by jon-flowers** [comment]
@ahmad-fakhrozy see this comment here:

https://github.com/tfutils/tfenv/issues/319#issuecomment-1045215942

Hashicorp didn't release ARM-specific versions until 1.0.2, so even though TFENV supports ARM with the TFENV_ARCH=arm64 setting, you have to actually work around it to use versions earlier than 1.0.2.

Basically, if you set that variable to amd64 instead, it will download successfully, and _should_ work fine.  I have done this with .12.x and .13.x without issues

Basically, try: 
`TFENV_ARCH=amd64 tfenv install 0.14.9`

**Comment by ahmad-fakhrozy** [comment]
thank you!
